berkeley-mobile/Drawer/MainDrawerViewController.swift

```diff
@@ -14,7 +14,7 @@ class MainDrawerViewController: DrawerViewController {
     var bottomOffset: CGFloat? {
         didSet {
             tabBarViewController.view.frame = self.view.bounds.inset(by: UIEdgeInsets(top: 0, left: 0, bottom: bottomOffset! + 20, right: 0))
-            tabBarViewController.view.frame.origin.y = barView.frame.maxY + 16
+            tabBarViewController.view.frame.origin.y = barView.frame.maxY + 24
```

Comment:
This seems to remove the small bit of bottom padding originally below the table views. Could you adjust the frame of `tabBarViewController` (in the line above this one) to re-add the padding?
<img width="559" alt="Screen Shot 2020-10-04 at 4 41 33 PM" src="https://user-images.githubusercontent.com/41145903/95029911-8f8de400-0660-11eb-864e-0ffc88c02671.png">